rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/packagespecific/NupkgPackageSpecificSpec.groovy

```diff
@@ -0,0 +1,75 @@
+package com.netflix.spinnaker.rosco.providers.util.packagespecific
+
+import com.netflix.spinnaker.rosco.providers.util.PackageNameConverter
+import com.netflix.spinnaker.rosco.providers.util.TestDefaults
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class NupkgPackageSpecificSpec extends Specification implements TestDefaults {
+    @Unroll
+    void "nupkg package names are properly parsed"() {
+        when:
```

Comment:
Please un-indent the `when:`, `then:`, and `where:` lines by 2 spaces so their children show as indented.